python/Produce_Config_File.py

```diff
@@ -86,24 +86,23 @@ def CyclicColor(DrawArg):
 	text_file.write("\t\t\tPlot_Name = '%s';\n"%(str(legEntry)))
 	text_file.write("\t\t\t[BEGIN_DATA]\n" )
 	text_file.write("\t\t\tVAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR\n" )
-	DrawArg=DrawArg+1
 
 	for row in range(args.SelectRowStart,args.SelectRowEnd):
-		if bool(args.YaxisScale):
+		if args.YaxisScale=='true':
```

Comment:
This is bad design.  If the user calls the help menu:

```
If YaxisScale option is true the Y axis is plotted in kUnit. Default is: False"
```

This is reported, this leads the user to think this is boolean.  Additionally string comparison like this will fail for:

```
True
tRue
trUe
truE
TRUE
etc...
```

Change back to boolean, make the action `store_true` so the user just has to call the option if it is desired, and change the if statements to be cleaner, e.g. the first case I showed above.